WcaOnRails/db/migrate/20160731000001_create_committees.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+class CreateCommittees < ActiveRecord::Migration
+  def change
+    create_table :committees do |t|
+      t.string :name, limit: 50, null: false
+      t.string :slug, limit: 50, null: false
```

Comment:
Yes millions of committees would certainly be a problem :)

As these are varchar fields they are variable length so there is no space saving by limiting the length. I think it's good practice though since it ensures database integrity. Is the rails application the only way we'll ever insert data? If we want to use a different tool to do reporting then having a sensible length set in the database helps understand the size of the data. If later on we want to extend the length this is easily done. To me it makes good sense to have limits in the database.

It is important to have consistency in projects so I have no issue in removing these limits as you suggested since that's how it is being done for all other migrations.